src/test/java/org/ibp/api/java/impl/middleware/inventory/LotMergeValidatorTest.java

```diff
@@ -0,0 +1,163 @@
+package org.ibp.api.java.impl.middleware.inventory;
+
+import org.generationcp.middleware.domain.inventory.manager.ExtendedLotDto;
+import org.generationcp.middleware.domain.inventory.manager.LotMergeRequestDto;
+import org.generationcp.middleware.pojos.ims.LotStatus;
+import org.ibp.api.exception.ApiRequestValidationException;
+import org.ibp.api.java.impl.middleware.inventory.manager.validator.LotMergeValidator;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@RunWith(MockitoJUnitRunner.class)
+public class LotMergeValidatorTest {
+
+	private static final String LOT_KEEP_UUID = "lotUUID";
```

Comment:
I prefer to use RandomStringUtils, to we ensure the tests pass with any input @lik-83 
In general, again add finals

Reply:
Done!